torch/_inductor/codegen/triton.py

```diff
@@ -2600,8 +2601,9 @@ def _mask_value(value, default) -> CSEVariable:
                 peer_val = self.codegen_cooperative_reduction_peer_combine(
                     f"{result_var}_bval", src_dtype, default
                 )
+                index_dtype = self.features.select_index_dtype()
                 peer_idx = self.codegen_cooperative_reduction_peer_combine(
-                    result_var, dtype, 0
+                    result_var, index_dtype, torch.iinfo(index_dtype).max
```

Comment:
is it on purpose to change the default value from 0 to torch.iinfo(index_dtype).max ?

Reply:
Yes, this is to match what happens above and fix a minor correctness issue when you have a tensor full of `inf` (and no other values).